multitenancy/src/main/java/org/axonframework/extensions/multitenancy/components/MultiTenantSubscribableMessageSource.java

```diff
@@ -0,0 +1,27 @@
+package org.axonframework.extensions.multitenancy.components;
```

Comment:
Since we've made this `SubscribableMessageSource` interface `EventMessage`-specific, I think we should move it to an event-specific folder.

I know that's a different placement comparted to the `SubscribableMessageSource` itself, but the base interface does not dictate a message type, whereas this one does.